### DISCUSSION TITLE
Voice Cloning - Urdu - Urdu

### DISCUSSION THREAD

**Discussion by dev1042** [coqui-ai/TTS#2766] [General Q&A]
I admire your exceptional work. I have a quick query, Although I understand Urdu TTS is not currently supported, I was wondering if there is any assistance available for cloning a voice in Urdu-Urdu speech. I aim to improve speech accent & pronunciation and speech quality. Any guidance or support would be greatly appreciated.

I have tried free vc to clone voice using urdu speech audio but the output audio is missing few words and not up to the mark voice synthesis/cloning.

**Comment by erogol** [comment]
https://tts.readthedocs.io/en/latest/inference.html#example-text-to-speech-using-fairseq-models-in-1100-languages

**Comment by Aljannat-Dev** [comment]
@dev1042 DO U MANAGED TO FIND ANY BEST SOLUTIUON FOR URDU TEXT TO SPEECH GENERATION IN NATURAL SOUNDING VOICE ?WITH OR WITHOUT VOICE CLONNING ABILITY?\
